src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let mens = people.filter(item => item.sex === 'm');
+
+  mens = !century ? mens
+    : mens.filter(item => {
+      return Math.ceil(item.died / 100) === century;
+    });
+
+  const averageMensAge = mens.reduce((acc, { died, born }) =>
+    acc + died - born, 0) / mens.length;
```

Comment:
We use this piece of code several times, so it would be nice to extract it into a helper function.